support-config/src/main/resources/touchpoint.PROD.conf

```diff
@@ -73,5 +73,8 @@ touchpoint.backend.environments {
         campaigns = "MembershipSub-Campaigns-PROD"
       }
     }
+    paper-round-api {
+      url = "https://testapi.guardianhnd.co.uk/v1/guardian" // api key in parameter store
```

Comment:
I assume it's correct that these are the same currently?

Reply:
It is: at the moment we’re pointing to the test API for everything because the production API isn’t ready. We have a Trello card in the backlog to switch to their production API before we go live

Reply:
(Which I will need to update now the config is moving)